network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -194,7 +194,6 @@ class BlogPage(FoundationMetadataPageMixin, Page):
         FieldPanel('first_published_at'),
         PrivacyModalPanel(),
     ]
-
     translatable_fields = [
```

Comment:
**Nitpick** I think the blank lines between these long lists help readability.

Reply:
Updated! Marking as resolved 👍